esmvaltool/recipes/examples/recipe_check_obs.yml

```diff
@@ -244,6 +244,62 @@ diagnostics:
          type: reanaly, version: 1, start_year: 1948, end_year: 2018}
     scripts: null
 
+  OSI-450:
+    description: OSI 450 monthly data
+    variables:
+      sic:
+        preproc: nopp
```

Comment:
```suggestion
```
I think this preprocessor has been removed from the recipe, because it was the default one anyway